### DISCUSSION TITLE
Umfrage als Ampel

### DISCUSSION THREAD

**Discussion by MT-private** [Wendelpunkt/ToypadChallenge#31] [Ideas]
Zur Zufriedenheitsumfrage kann man eine Ampel leuchten lassen und jeder stellt sein Männchen auf das entsprechende Feld. Im Launcher erscheint dann eine grafische Darstellung der Umfrage.
Im Kindergarten meiner Tochter wird so abgefragt, wie das Essen geschmeckt hat. Sie müssen sich selbst vor nen Smiley stellen.

**Habe ich bereits implementiert und als PR gepusht.** 

Möchte jetzt noch Figuren bauen oder unter PeppaPig- Figuren aus Pudding-Eiern Tags kleben. Ist zwar kein Lego, aber unsere Kinder lieben die überteuerten Pudding-Eier und so lohnt der Kauf. Und so kann ich dann auch sagen "Alle lieben Pudding-Eier" (so hört gefühlt jede Peppa-Folge auf.)

**Comment by tomwendel** [comment]
Oh Mist. Die Notification im Weihnachtstrubel übersehen. Da schau ich doch morgen gleich noch rein. Vielen Dank schon mal fürs Mitmachen!

**Reply by MT-private** [reply]
Gerne und Danke für das Feedback.
Aktuell ist es so implementiert, dass man seine Meinung auch ändern könnte. In die Statistik fließt immer nur die letzte Wertung ein. Eine Sperre, dass man seine Meinung ändern könnte, ließe sich leicht einbauen.

Grüße und nen guten Start in 2025

**Reply by tomwendel** [reply]
Oh. Dann habe ich die Implementierung beim Überfliegen falsch interpretiert. "Meinung ändern" scheint mir hier das richtige und erwartete Verhalten zu sein. Daher: Alles gut. ich will nix gesagt haben :D
